src/index.html

```diff
@@ -3689,6 +3689,7 @@ <h4>Indicator chart title</h4>
       }
     });
   </script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jspdf/1.3.4/jspdf.debug.js"></script>
```

Comment:
how is this added to the html? I don't see import-webflow-export import code to add it.

Reply:
@waracci has this been resolved? I don't want to merge the code if we're adding something that will be clobbered when Matt exports again from webflow